fbpic/fields/interpolation_grid.py

```diff
@@ -70,18 +70,6 @@ def __init__(self, Nz, Nr, m, zmin, zmax, rmax, use_cuda=False ) :
         # Note: No Verboncoeur-type correction required
         self.invvol = 1./vol
```

Comment:
Could you update the docstring of the `InterpolationGrid` to say that this is a base class, which is used for the `FieldInterpolationGrid` and `EnvelopeInterpolationGrid`?